packages/nocodb/src/modules/jobs/jobs/export-import/duplicate.processor.ts

```
@@ -35,112 +35,151 @@
+  }
+
+  async duplicateBase(job: Job<DuplicateBaseJobData>) {
+    this.debugLog(`job started for ${job.id} (${JobTypes.DuplicateBase})`);
+
+    const { context, sourceId, dupProjectId, req, options } = job.data;
+
+    const baseId = context.base_id;
+
+    // workspace templates placeholder user
+    if (req.user?.id === '1') {
+      delete req.user;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)